edgexfoundry/device-usb-camera
# build: Update to use latest sdk

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

- make docker
- /compose-builder$ make gen no-secty ds-usb-camera (this generates docker-compose.yml with latest edgex images)
- Replace usb docker image in docker-compose.yml with your latest image created using `make docker` from the previous step
- /compose-builder$ make up
- usb devices should be registered in Edgex

## New Dependency Instructions (If applicable)
